andela/travel_tool_front
# #160815252 Mark single notification as read

#### What does this PR do?
Marks a single notification as read when details of the notification is viewed

#### Description of Task to be completed?
- dispatch action to mark a notification as read
- change the state of the `notificationStatus` from `unread` to `read`
- make api call to update the notification status

#### How should this be manually tested?
- Clone the repo
- Change into the project directory
- Check out this branch with `git checkout ft-mark-notification-as-read-160815252`
- Start the server `make start`
- Setup the backend by following this [PR](https://github.com/andela/travel_tool_back/pull/82)
- In a browser, navigate to `localhost:3000`
- Toggle open the notification pane
- Click on `View Details` link on a notification
- Close the request details modal
- You should see visual indications that the notification has been read.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160815252](https://www.pivotaltracker.com/story/show/160815252)

#### Screenshots (if appropriate)
None

#### Questions:
None
